### ISSUE TITLE
Allow interacting with devices while sitting in a chair

### ISSUE THREAD

**Issue by Regalis11** [Regalis11/Barotrauma#3033]
@Regalis11 commented on [Wed Jun 03 2020](https://github.com/Regalis11/Barotrauma-development/issues/1930)

Related to https://github.com/Regalis11/Barotrauma/issues/5587

**Comment by NilanthAnimosus** [comment]
Tested in 0.19.0.0

I've been testing this for awhile, and finally found an issue with AI pathing involving chairs:

![image](https://user-images.githubusercontent.com/29177976/181444023-918f5861-cf50-4d48-94f4-b49260149186.png)

The AI was unable to stand up, cancelling and restarting the order he was stuck to the reactor. I didn't order him to the reactor (he sat doing an automatic order to activate it at round start leaving the outpost or just when he was idling) then I used an explosion with EMP to damage a junction which I wanted to quickly test to find he didn't begin moving, it could be due to the placement of the junction or the seat near the reactor...I'm not sure which it is.

teleporting him out of the chair he ran to do his repair order as normal.

I tried the same with cleanup items and find safety getting problems with pathfinding. it appears if a new order appears and they need to pathfind to it - they might get stuck in the chairs if their already sitting in them.

submarine edited and used:
[Winterhalter - chair test.zip](https://github.com/Regalis11/Barotrauma/files/9206520/Winterhalter.-.chair.test.zip)
